AWS/README.md

```diff
@@ -38,50 +38,337 @@
  # AWS Actions 
 
 * Aws apply default encryption for s3 bucket.py
+* Aws attach ebs to instances.py
```

Comment:
Are these Links? I am not seeing that in here. Appears like a file name (First letter capitalized).py

Reply:
this was a bug in the Script - its fixed now